### ISSUE TITLE
kill-tabs stopeed working with npm@3.4.1

### ISSUE THREAD

**Issue by tomalec** [sindresorhus/kill-tabs#10]
I have no idea what else could I write here :/
I have just realized after few npm & node updates that `kill-tabs` stopped working.
It simply does nothing, none of tabs get's killed, no error, no message gets shown.

``` sh
$ node --version
v4.2.2

$ npm --version
3.4.1

$ kill-tabs

$
```

I'm using Windows 8.1, and I have tried re-installing many times.

**Comment by sindresorhus** [comment]
Are you using the latest version of `kill-tabs`? Maybe try reinstalling and make sure nothing got corrupted? npm has a tendency to do that.

**Comment by tomalec** [comment]
```
$ kill-tabs --version
2.0.0
```

I did reinstall, how can I make sure? No error occurred during install.

**Comment by sindresorhus** [comment]
Fixed in 2.0.1.
